excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod

```diff
@@ -112,7 +115,11 @@ Ps0 f11 actionCode 'import com.axonivy.connector.excel.demo.Person;
 out.exportRecordset = new Recordset();
 
 for (Person p : in.persons){
- Record r = new Record(["firstname","lastname"], [p.firstname,p.lastname]);
+ List <Object> values = new List <Object>();
+ for (String key : in.recordsetKeys){
+  values.add(p.get(key));
+ }
+ Record r = new Record(in.recordsetKeys, values);
```

Comment:
working with records (=rows) limits the capability of excel where you can have more than one type in a row.